internal/store/store.go

```diff
@@ -0,0 +1,52 @@
+package store
+
+import (
+	"context"
+
+	"cloud.google.com/go/firestore"
+	"github.com/covid19cz/erouska-backend/internal/constants"
+	"github.com/covid19cz/erouska-backend/internal/firebase"
+)
+
+// Storer is a storage abstraction layer interface
+type Storer interface {
```

Comment:
Not sure if I like the name `Storer`... :thinking:

Reply:
interface in golang are afaik ending with -er, like Stringer, Reader, Writer, etc.
https://stackoverflow.com/questions/38842457/interface-naming-convention-golang

Reply:
OK then.